Mlem/App/Views/Pages/MessageFeedView/MessageFeedView.swift

```diff
@@ -0,0 +1,220 @@
+                            NavigationLink(.person(person)) {
+                                HStack(spacing: Constants.main.halfSpacing) {
+                                    CircleCroppedImageView(person, frame: 24)
+                                    Text(person.displayName)
+                                        .foregroundStyle(palette.primary)
+                                        .font(.headline)
+                                    Image(systemName: Icons.forward)
+                                        .imageScale(.small)
+                                        .fontWeight(.semibold)
+                                        .foregroundStyle(palette.tertiary)
+                                }
+                            }
+                        }
+                        ToolbarItemGroup(placement: .secondaryAction) {
+                            SwiftUI.Section {
+                                if person is any Person3Providing, proxy.isLoading {
+                                    ProgressView()
+                                } else {
+                                    MenuButtons {
+                                        person.menuActions(
+                                            isInMessageFeed: true,
+                                            navigation: navigation,
+                                            community: nil
+                                        )
+                                    }
+                                }
+                            }
+                        }
+                    }
+            }
+        }
+    }
+    
+    @ViewBuilder
+    func content(person: any Person) -> some View {
+        ScrollViewReader { scrollProxy in
+            ScrollView {
+                if let feedLoader {
+                    LazyVStack(spacing: 0) {
+                        ForEach(feedLoader.items.reversed()) { message in
+                            if !message.deleted {
+                                bubbleView(message: message, feedLoader: feedLoader)
+                                    .id(message.id)
+                            }
+                        }
+                    }
+                    .scrollTargetLayout()
+                    .padding(.top, 50)
+                }
+            }
+            .safeAreaInset(edge: .bottom) {
+                textInput(scrollProxy)
+            }
+            .defaultScrollAnchor(.bottom)
+            .scrollDismissesKeyboard(.interactively)
+            .background(palette.groupedBackground)
+            .onAppear {
+                if feedLoader == nil {
+                    feedLoader = .init(person: person, pageSize: 50)
+                    Task { @MainActor in
+                        do {
+                            try await feedLoader?.loadMoreItems()
+                        } catch {
+                            handleError(error)
+                        }
+                    }
```

Comment:
What's the reason for this being here instead of `.init()`?

Reply:
The feedloader can't be made if we've only got a `PersonStub` - it requires a `Person`.